core/src/main/java/dev/jdtech/jellyfin/utils/DownloaderImpl.kt

```diff
@@ -82,20 +85,40 @@ class DownloaderImpl(
                     if (intro != null) {
                         database.insertIntro(intro.toIntroDto(item.id))
                     }
-                    val request = DownloadManager.Request(source.path.toUri())
-                        .setTitle(item.name)
-                        .setAllowedOverMetered(appPreferences.downloadOverMobileData)
-                        .setAllowedOverRoaming(appPreferences.downloadWhenRoaming)
-                        .setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED)
-                        .setDestinationUri(path)
-                    val downloadId = downloadManager.enqueue(request)
-                    database.setSourceDownloadId(source.id, downloadId)
-                    return Pair(downloadId, null)
+                    if (appPreferences.downloadQuality != "Original") {
```

Comment:
This string is a sort of [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) (i'm not sure if there's a better term which also encompasses strings). You should probably replace this with an enum